packages/ui/src/components/YouVersionAuthButton.tsx

```diff
@@ -136,10 +152,25 @@ export const SignInButton = React.forwardRef<HTMLButtonElement, SignInButtonProp
 
     const buttonLoading = auth.isLoading;
 
-    let buttonCopy = 'Sign in with YouVersion';
-    if (size === 'short') {
-      buttonCopy = 'Sign in';
-    }
+    const buttonText = useMemo(() => {
+      if (size === 'short') {
+        if (mode === 'signOut') {
+          return 'Sign Out';
+        } else if (mode === 'signIn') {
+          return 'Sign In ';
```

Comment:
Remove trailing space.

```suggestion
          return 'Sign In';
```